pylib/spinnaker/change_cassandra.py

```diff
@@ -104,12 +104,19 @@ def __init__(self, options):
     self.__bindings.import_dict(config)
 
   def disable_cassandra(self):
+    if os.path.exists('/opt/spinnaker/cassandra/SPINNAKER_INSTALLED_CASSANDRA'):
+      os.remove('/opt/spinnaker/cassandra/SPINNAKER_INSTALLED_CASSANDRA')
+      open('/opt/spinnaker/cassandra/SPINNAKER_DISABLED_CASSANDRA', 'w').close()
     with open('/etc/init/cassandra.override', 'w') as f:
         f.write('manual')
     print 'Stopping cassandra service...'
     os.system('service cassandra stop')
 
   def enable_cassandra(self):
+    if os.path.exists('/opt/spinnaker/cassandra/SPINNAKER_DISABLED_CASSANDRA'):
+      os.remove('/opt/spinnaker/cassandra/SPINNAKER_DISABLED_CASSANDRA')
+      (open('/opt/spinnaker/cassandra/SPINNAKER_INSTALLED_CASSANDRA', 'w')
```

Comment:
What's the reason this line has parens around it and the similar line in `disable_cassandra()` does not?